readme.md

```diff
@@ -27,13 +27,13 @@ Your app must...
 * ...**have complete RESTful routes** for at least one of your resources with GET, POST, PUT, and DELETE.
 * ...**utilize an ORM to create a database table structure** (e.g. ActiveRecord) and interact with your relationally-stored data.
 * ...have **semantically-clean HTML and CSS that passes the standard validators**. (See "Resources" below)
-* ...**be deployed online** and accessible to the public. ***We will be having a mandatory lesson on deployment Tuesday morning at 10:00am***
+* ...**be deployed online** and accessible to the public.
```

Comment:
So we either (a) need to have a mandatory mini-lesson on deployment or (b) remove deployment as a requirement.